blog/_posts/2021-07-31-Mitchell-Johnson_Jamie-Schmidt.md

```diff
@@ -0,0 +1,118 @@
+---
+layout: topic
+title: "Development of MABE2"
+date: 2021-07-31
+author: Mitchell Johnson & Jamie Schmidt
+---
+
+### Development of MABE2
```

Comment:
The title will be automatically generated by Jekyll so you don’t have to put it in manually

Reply:
removed